src/Generator.php

```diff
@@ -1,5 +1,4 @@
 <?php
-declare(strict_types=1);
 
```

Comment:
There shouldn't be a blank line between the opening PHP tag and the license header, as per my previous comment.